app/src/main/java/camp/nextstep/edu/calculator/MainActivity.kt

```diff
@@ -11,5 +11,18 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
+
+        binding.button0.setOnClickListener{ binding.textView.text = "0" }
```

Comment:
```suggestion
        binding.button0.setOnClickListener{ binding.textView.text = binding.button0.text.toString() }
```
지금도 잘 작성해주셨지만, 버튼의 텍스트를 활용해보면 어떨까요? (이하 관련 내용 동일)

Reply:
text 입력 값을 그대로 사용할 수 있군요!